internal/kafka/kafka_rest_onprem.go

```diff
@@ -21,6 +21,11 @@ func initKafkaRest(c *pcmd.AuthenticatedCLICommand, cmd *cobra.Command) (*kafkar
 	if err != nil { // require the flag
 		return nil, nil, "", err
 	}
+
+	if strings.Contains(url, "confluent.cloud") {
```

Comment:
What about calling `ccloudv2.IsCCloudURL(url, false)` instead? Would also cover testing and gov environments.

Reply:
Good idea, adding!